Dockerfile.tools

```diff
@@ -26,9 +26,9 @@ RUN mkdir -p $HOME && \
     tar -C /usr/local -xzf go.tar.gz && \
     rm go.tar.gz && \
     # get required golang tools and OC client
-    go get github.com/onsi/ginkgo/ginkgo && \
-    go get golang.org/x/lint/golint && \
-    go get github.com/mattn/goveralls && \
+    go install github.com/onsi/ginkgo/ginkgo@latest && \
+    go install golang.org/x/lint/golint && \
```

Comment:
you need to use `@latest` also here (and below)
Additionally, because of how prow works now, the better sequence seems to be:

1. send a PR to fix `Dockerfile.tools`, so the prow jobs build using prow. Can be worth emit the output of `go version` when building to easily doublecheck
2. send a PR to fix the github actions
3. only once both PRs above are merged ok, send a PR to bump the go version in `go.mod` (and related like `go.sum`, vendor...)

Reply:
done, thx